### PR TITLE
indicate ongoing upgrade

### PR DESCRIPTION
**Pull request by MaximilianMeister**
this makes it easier for the crowbar api to determine if the upgrade
is running.
also use a variable for the crowbar install directory.

### CODE REVIEW

File: scripts/upgrade_admin_server.sh [hunk 1]
```diff
@@ -20,6 +20,16 @@ set -x
 
 upgrade_admin_server()
 {
+    local installdir=/var/lib/crowbar/install
+
+    if [[ -f $installdir/admin_server_upgrading ]] ; then
+        echo "Exit: Upgrade already running..."
+        exit 1
```
Comment (jsuchome) [inline scripts/upgrade_admin_server.sh]: As this function is executed in background, we will  not know the exit value. So the check if the upgrade is already running should be done directly from crowbar, I think.

**Comment by jsuchome** [comment]
+1

File: scripts/upgrade_admin_server.sh (hunk 1 above)
Comment (MaximilianMeister) [inline scripts/upgrade_admin_server.sh]: right, noted

Comment (MaximilianMeister) [inline scripts/upgrade_admin_server.sh]: https://github.com/crowbar/crowbar-core/pull/604/commits/092b2ae40161a519a722a202e0e3f88804226c8a

Comment (jsuchome) [inline scripts/upgrade_admin_server.sh]: Nice

**Comment by rsalevsky** [comment]
+1

**Comment by jsuchome** [comment]
Please backport to 3.0

**Pull request merged**